data-management/data-interface.js

```diff
@@ -32,7 +32,7 @@ async function validateInputArms(userID, accessList, accessStatuses) {
 async function checkAdminPermissions(userInfo) {
     let result = await neo4j.getMyUser(userInfo);
     try{
-        return result.role === 'admin' && result.userStatus === 'active';
+        return result.role === ADMIN && result.userStatus === ACTIVE;

```

Comment:
The user role is stored as 'admin' currently. This wont work with 'Admin'. Active as well. Please, use isCaseInsensitiveEqual(result.role, ADMIN) && isCaseInsensitiveEqual(result.userStatus, ACTIVE);

This PR will also affect the ticket for revoking admin feature.